runconfig/hostconfig_windows.go

```diff
@@ -71,3 +70,15 @@ func ValidateResources(hc *container.HostConfig, si *sysinfo.SysInfo) error {
 	}
 	return nil
 }
+
+// ValidatePrivileged performs platform specific validation of the Privileged setting
+func ValidatePrivileged(hc *container.HostConfig) error {
```

Comment:
Looks like this function doesn't have to be exported (I realize some of the others probably don't have to be exported as well)